src/NzbDrone.Common/Instrumentation/Sentry/SentryDebounce.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NzbDrone.Common.Instrumentation.Sentry
+{
+    public class SentryDebounce
+    {
+        private readonly Dictionary<string, DateTime> _dictionary;
```

Comment:
Why not use Cached? It has a built in expiry iirc.

Also, maybe have a counter on the number of events? If that's relevant to Sentry. (Not a priority, do it later when it's stabilized)